src/form.py

```diff
@@ -357,6 +397,8 @@ def handle_release_knob_changed(self, value) -> None:
     def handle_pitch_knob_changed(self) -> None:
         """
         This function handles when the pitch knob value is changed.
+        Args:
+        value: the number setting for the pitch knob
```

Comment:
this function doesn't have a value argument

Reply:
nice catch, I've updated this docstring